### DISCUSSION TITLE
Can't install Ubuntu 24.04 LTS on Orange pi 5 Plus

### DISCUSSION THREAD

**Discussion by Lolmc0587** [Joshua-Riek/ubuntu-rockchip#1156] [Q&A] [answered]
![IMG_2736](https://github.com/user-attachments/assets/b595c18f-2c35-4201-8a59-1c9500fcc72f)

i have copied the error log here:
`dpkg: error processing package grub-efi-arm64-bin (-remove): unable to securely remove*/usr/share/bug/grub-efi-arm64-bin/script.dpkg-tmp': Bad message`

**Comment by hakotani-o** [comment]
Hi Lolmc0587.

For what purpose are you trying to install grub?

Thank you.

**Reply by Lolmc0587** [reply]
I mean that error occurred when i trying to install new os
That is the error i copied from that image

**Reply by hakotani-o** [reply]
I will create a disk image of Orangepi-5-plus and test it.

**Reply by hakotani-o** [reply]
I confirmed that it can be installed by writing to the SD-CARD.

ubuntu-24.04-preinstalled-desktop-arm64-orangepi-5-plus.img.xz.sha256
https://drive.google.com/file/d/1DdE-T24NxoTHlXc4tCuELi2_t-SAPhzL/view?usp=drive_link

ubuntu-24.04-preinstalled-desktop-arm64-orangepi-5-plus.img.xz
https://drive.google.com/file/d/1cJjNAekmtGUs9IrbWTHoa1Sb5VxTrcF6/view?usp=drive_link

**Reply by Lolmc0587** [reply, marked as answer]
i have found this version work 
https://github.com/Joshua-Riek/ubuntu-rockchip/releases/tag/v2.2.1

Thanks for your help!
